scripts/20210616_remove-permissionList-read/remove-permissionList-read.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
+
+// Number of forms with permissionList.read key - expect 0 after running update
+db.forms.count({
+  permissionList: { $elemMatch: { read: { $exists: true } } }
```

Comment:
no need to change but this is equivalent to `'permissionList.read': { $exists: true }`